tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
```

Comment:
use newer target(es2019/es2020)
when you use target `es5` you shuld not use Promises at all since Promise exist in `js` only from `es6`

Reply:
done